roofit/multiprocess/inc/RooFit/MultiProcess/HeatmapAnalyzer.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   ZW, Zef Wolffs, Nikhef, zefwolffs@gmail.com
+ *
+ * Copyright (c) 2022, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOT_ROOFIT_MultiProcess_HeatmapAnalyzer
+#define ROOT_ROOFIT_MultiProcess_HeatmapAnalyzer
+
+#include "TString.h"
+#include "TH2I.h"
+
+#include <vector>
+#include <string>
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
+
+namespace RooFit {
+namespace MultiProcess {
+
+class HeatmapAnalyzer {
+public:
+   HeatmapAnalyzer(TString logs_dir);
```

Comment:
```suggestion
   HeatmapAnalyzer(std::string const& logs_dir);
```
Try to avoid using the old `TString` please